### ISSUE TITLE
Uncaught Error: Cannot decorate a destroyed marker

### ISSUE THREAD

**Issue by Sod-Almighty** [sirbrillig/mixed-indent-warning#19]
**Atom**: 1.25.0 x64
**Electron**: 1.7.11
**OS**: Mac OS X 10.11.6
**Thrown From**: [mixed-indent-warning](https://github.com/sirbrillig/mixed-indent-warning) package 0.3.0

### Stack Trace

Uncaught Error: Cannot decorate a destroyed marker

```
At /Applications/Atom.app/Contents/Resources/app/src/decoration-manager.js:191

Error: Cannot decorate a destroyed marker
    at DecorationManager.decorateMarker (/Applications/Atom.app/Contents/Resources/app/src/decoration-manager.js:183:27)
    at TextEditor.decorateMarker (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:2137:41)
    at /packages/mixed-indent-warning/lib/mixed-indent-warning.coffee:51:14
    at Array.forEach (native)
    at Object.scanFile (/packages/mixed-indent-warning/lib/mixed-indent-warning.coffee:47:21)
    at /packages/mixed-indent-warning/lib/mixed-indent-warning.coffee:32:14
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at TextBuffer.module.exports.TextBuffer.emitDidStopChangingEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/text-buffer.js:1773:26)
    at later (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/helpers.js:28:13)
```

### Commands

```
     -2:11.8.0 core:confirm (input.hidden-input)
  2x -1:58.5.0 core:close (button.btn.icon.enablement.icon-playback-play)
     -1:53.1.0 project-find:show (input.hidden-input)
     -1:52.6.0 core:confirm (input.hidden-input)
     -1:36.7.0 core:move-up (input.hidden-input)
     -1:36.6.0 core:move-down (input.hidden-input)
     -1:36.3.0 core:move-up (input.hidden-input)
     -1:36.2.0 core:move-down (input.hidden-input)
     -1:35.8.0 pane:show-next-recently-used-item (input.hidden-input)
     -1:35.3.0 pane:move-active-item-to-top-of-stack (div.results-view.focusable-panel)
     -1:34.8.0 core:close (div.results-view.focusable-panel)
     -1:16.9.0 pane:show-next-recently-used-item (input.hidden-input)
     -1:16.7.0 pane:move-active-item-to-top-of-stack (input.hidden-input)
     -0:21.4.0 bookmarks:toggle-bookmark (input.hidden-input)
     -0:10.6.0 core:move-left (input.hidden-input)
     -0:09.8.0 core:move-right (input.hidden-input)
```

### Non-Core Packages

```
arpeggio 1.1.2 
atom-bracket-highlight 0.3.2 
atom-brackets-glow 0.6.0 
atom-ctags 5.1.2 
atom-cursor-indent 0.3.0 
atom-like-brackets-editor 1.2.0 
atom-material-syntax 1.0.8 
atom-material-ui 2.1.3 
auto-detect-indentation 1.3.0 
autocomplete-clang 0.11.4 
autocomplete-ruby 0.2.8 
bracket-matcher-highlight 0.5.0 
busy-signal 1.4.3 
color-picker 2.2.5 
custom-folds 1.9.5 
export-html 3.1.0 
file-icons 2.1.18 
file-types 0.5.5 
fold-comments 0.6.0 
fold-functions 0.5.0 
fold-lines 0.2.2 
ftp-remote-edit 0.12.12 
fun-files 0.10.0 
grammar-token-limit 0.1.1 
hex-editor 0.1.1 
highlight-bad-chars 1.0.7 
highlight-column 0.5.4 
indent-detective 0.2.0 
indent-guide-improved 1.4.13 
intentions 1.1.5 
jumpy 4.2.0 
language-dylan 0.7.3 
language-haml 0.31.0 
language-ini 1.19.0 
language-lisp 0.2.0 
language-livescript 0.0.3 
language-ruby-without-erb 0.60.0 
language-scheme 1.4.0 
line-ending-converter 1.3.2 
linter 2.2.0 
linter-clang 4.1.2 
linter-gcc 0.7.1 
linter-mixed-indent 1.1.1 
linter-ruby 1.2.6 
linter-ui-default 1.7.1 
list-edit 1.0.1 
menu-editor 0.2.0 
mixed-indent-warning 0.3.0 
multi-cursor-plus 1.2.0 
navigation-history 0.1.8 
parinfer 1.22.0 
permanent-delete 0.1.7 
pigments 0.40.2 
preview 0.17.0 
project-folder 1.5.0 
project-plus 1.0.0 
rainbow-delimiters 2.1.2 
rainbow-tabs 2.5.1 
regex-railroad-diagram 0.19.4 
remember-folds 0.3.0 
remote-sync 4.1.8 
ruby-block 0.3.6 
ruby-block-converter 4.0.1 
ruby-navigator 0.5.0 
sane-indentation 0.2.1 
set-indent 0.1.1 
show-invisibles-plus 0.3.1 
simple-drag-drop-text 0.4.0 
space-tab 0.4.2 
spacecase 0.1.0 
Stylus 3.1.1 
swackets 0.35.0 
switch-header-source 0.28.0 
sync-settings 0.8.6 
tab-control 0.6.10 
tab-limiter 2.2.0 
tab-switcher 1.5.6 
tabs-to-spaces 1.0.3 
terminal-plus 0.14.5 
text-pastry 0.6.1 
toggle-gutter 0.3.4 
tool-bar 1.1.0 
trailing-spaces 0.4.0 
tree-ignore 0.6.2 
tree-view-breadcrumb 0.8.1 
```

**Comment by Arcanemagus** [comment]
Looks like this is from an older version of this package (v0.3.0), you'll want to update to the current version.

**Comment by sirbrillig** [comment]
Yup. This is fixed in version 1.0.
